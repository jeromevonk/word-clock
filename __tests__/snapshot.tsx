import { act, render } from '@testing-library/react'
import Home from '../pages/index'

it('renders homepage unchanged', () => {
  
  act(() => {
    const { container } = render(<Home />)
    expect(container).toMatchSnapshot()
  });
  
})
