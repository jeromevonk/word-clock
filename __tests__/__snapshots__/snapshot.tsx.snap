// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class="container"
  >
    <main>
      <div
        class="stackRow"
      >
        <div
          class="buttonMode"
        />
      </div>
    </main>
    <footer
      class="footer"
    >
      <a
        href="https://jeromevonk.github.io/"
        target="blank"
      >
        By Jerome Vonk
      </a>
    </footer>
  </div>
</div>
`;
